usecases/build/build.go

```diff
@@ -0,0 +1,55 @@
+//                           _       _
+// __      _____  __ ___   ___  __ _| |_ ___
+// \ \ /\ / / _ \/ _` \ \ / / |/ _` | __/ _ \
+//  \ V  V /  __/ (_| |\ V /| | (_| | ||  __/
+//   \_/\_/ \___|\__,_| \_/ |_|\__,_|\__\___|
+//
+//  Copyright © 2016 - 2024 Weaviate B.V. All rights reserved.
+//
+//  CONTACT: hello@weaviate.io
+//
+
+package build
+
+import (
+	"runtime"
+
+	"github.com/prometheus/common/version"
+)
+
+// These package level variables are set during build time via ldflags -X
+// Also these information are passed to prometheus version package so that it's available in `weaviate_build_info` metric
+var (
+	// Version is Weviate's version. e.g: 1.26.4.
+	// We usually read it from `openapi-specs/schema.json` in this repository.
+	Version string
+
+	// Revision represents the GIT commit of the build.
+	Revision string
```

Comment:
I'm thinking of be

Reply:
I don't have too much opinion here. Just inspired from prometheus version package.
https://github.com/prometheus/common/blob/main/version/info.go#L27-L37

Sounds good to change to `GitRevsion`, `GitBranch`.